Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -3301,6 +3299,12 @@ B3::ValueRep BBQJIT::toB3Rep(Location location)
 StackMap BBQJIT::makeStackMap(const ControlData& data, Stack& enclosingStack)
 {
     unsigned numElements = m_locals.size() + data.enclosedHeight() + data.argumentLocations().size();
+    if (Options::useWasmIPInt()) {
```

Comment:
I seem to recall that the LLInt->BBQ stackmap is re-used for BBQ->OMG, in which case this would be wrong. But then the code would already be very wrong, so I must be misremembering. Can you double check?

Reply:
OMG uses a different stackmap completely, in WasmIRGeneratorHelpers.h; so I don't believe this should affect BBQ->OMG.